src/types/check_config.rs

```diff
@@ -0,0 +1,27 @@
+use uuid::Uuid;
+
+/// Valid intervals between the checks.
+#[derive(Debug, Copy, Clone)]
+pub enum CheckInterval {
+    FiveMinutes = 5,
```

Comment:
Let's allow 1 minute as well, might be useful for our testing with rebalancing etc

Reply:
👍